tools/go-cli/go-whisk/whisk/shared.go

```diff
@@ -38,4 +38,5 @@ type Parameters *json.RawMessage
 type Limits struct {
     Timeout int `json:"timeout,omitempty"`
     Memory  int `json:"memory,omitempty"`
+    Logsize int `json:"logs"`
```

Comment:
The limits struct is instantiated when any of the memory, timeout or logsize arg is used.  Any arg not set will default to 0.  When the `omitempty` is removed, the default 0 value will be sent.  Can you try this test.
```
touch a.js
wsk action create tempaction a.js --logsize 10
wsk action get tempaction
wsk action update tempaction --memory 512  // leaving logsize alone
wsk action get tempaction    // Did the logsize setting change?
```